src/serverdlg.cpp

```diff
@@ -106,16 +106,16 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
     edtLocationCity->setAccessibleName ( tr ( "City where the server is located line edit" ) );
 
     // location country
-    QString strLocCountry = "<b>" + tr ( "Location country" ) + ":</b> " +
-                            tr ( "The country in "
-                                 "which this server is located can be set here. If a country is "
+    QString strLocCountry = "<b>" + tr ( "Country/Region" ) + ":</b> " +
+                            tr ( "The country or region in "
+                                 "which this server is located can be set here. If a location is "
                                  "entered, it will be shown in the connect dialog server list at the "
                                  "clients." );
```

Comment:
> As per style guide, we should try to put things into active case for better readability. So this would be:
>
> "Set the country or region in which this server is located here. Clients will show this location in their connect dialog's server list."

@gilgongo is this correct?

```suggestion
                            tr ( "Set the country or region in which this server is located here. "
                                 "Clients will show this location in their connect dialog's server "
                                 "list." );
```

Reply:
> server is located

"where the server is running" might be more explicit as to what's actually useful.

I've seen people putting the location the server is expected to be used from (e.g. "Cornwall" for a server that I'd say is almost certainly hosted in London, unless they've got a dedicated fibre from the data centre Jamulus Explorer runs in...  Bristol, which is closer and does have good network links, has a higher ping...).

Reply:
```suggestion
                            tr ( "Set the country or region where the server is running. "
                                 "Clients will show this location in their connect dialog's server "
                                 "list." );
```

@pljones is this correct?

Reply:
Looks good to me.